js/utilities.js

```diff
@@ -5,7 +5,7 @@ export const showToastMessage = (message) => {
   $toaster.hidden = false;
   setTimeout(() => {
     $toaster.hidden = true;
-  }, 3000);
+  }, 1500);
```

Comment:
It's better to not make this change to this PR. As it's was primarily done for the Add Tasks PR.

You may use something simple like appending a simple text as an alert, but it should not be DOM blocking, and you can always beautify the toaster in the UI design section.

Reply:
okay bhai